src/pages/communication-page/cat-a-mod1/communication.cat-a-mod1.page.ts

```diff
@@ -10,7 +10,7 @@ import { getCurrentTest, getJournalData } from '../../../modules/tests/tests.sel
 import { getTests } from '../../../modules/tests/tests.reducer';
 
 // TODO - PREP-AMOD1 - implement category specific reducer
```

Comment:
Can the TODO be removed with this change?

Reply:
Removing TODO